selfdrive/test/test_car_models.py

```diff
@@ -484,6 +484,7 @@ def get_route_log(route_name):
   TOYOTA.HIGHLANDER,
   TOYOTA.HIGHLANDERH,
   TOYOTA.HIGHLANDERH_TSS2,
+  TOYOTA.PRIUS_TSS2,
```

Comment:
we'll need a route before merging this

Reply:
Can you unban me so I can get route and then ban me back if you would like. I am just trying to upstream the fp.

Reply:
What's your dongle ID?